Client/Frontend/Browser/QRCodeViewController.swift

```diff
@@ -291,7 +291,9 @@ class QRCodeViewController: UIViewController {
         videoPreviewLayer.frame = UIScreen.main.bounds
         view.layer.addSublayer(videoPreviewLayer)
         self.videoPreviewLayer = videoPreviewLayer
-        captureSession.startRunning()
+        DispatchQueue.global(qos: .background).async {
```

Comment:
This is a user initiated action so shouldn't have a low priority. I would just set it to default tho which means you don't have to specify the parameter. We have many issues in the app at the moment because of threads with conflicting priorities so I think it's best to leave them at default unless there is a compelling reason to deviate.